modulefiles/build_derecho_intel.lua

```diff
@@ -6,15 +6,14 @@ the CISL machine Derecho (Cray) using Intel@2021.10.0
 whatis([===[Loads libraries needed for building the UFS SRW App on Derecho ]===])
 
 prepend_path("MODULEPATH","/lustre/desc1/scratch/epicufsrt/contrib/modulefiles_extra")
-prepend_path("MODULEPATH", "/glade/work/epicufsrt/contrib/spack-stack/derecho/spack-stack-1.5.1/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/glade/work/epicufsrt/contrib/spack-stack/derecho/spack-stack-1.7.0/envs/ue-intel/install/modulefiles/Core")
 
 load(pathJoin("stack-intel", os.getenv("stack_intel_ver") or "2021.10.0"))
 load(pathJoin("stack-cray-mpich", os.getenv("stack_cray_mpich_ver") or "8.1.25"))
-load(pathJoin("cmake", os.getenv("cmake_ver") or "3.26.3"))
+load(pathJoin("cmake", os.getenv("cmake_ver") or "3.23.1"))
```

Comment:
Is there a reason for this CMake version downgrade?

Reply:
@mkavulich, just used cmake build by spack-stack vs. using one built by admins. It was to be consistent with other machines which use spack-stack built cmake (3.23.1).
For Derecho, we can go with 3.26.3 if needed.

Reply:
Thanks for the explanation, I didn't realize we were previously using the platform build instead of through spack.